examples/plug-and-play/demo_PnP_mirror_descent.py

```diff
@@ -0,0 +1,122 @@
+r"""
+Plug-and-Play algorithm with Mirror Descent for Poisson noise inverse problems.
+====================================================================================================
+
+This is a simple example to show how to use a mirror descent Plug-and-Play algorithm for solving an inverse problem with Poisson noise.
+The prior term is a RED denoising prior with DnCNN denoiser.
+"""
```

Comment:
maybe here you can explain a bit more about mirror descent, and write the iteration?